form.yml

```diff
@@ -13,30 +13,25 @@ attributes:
     widget: select
     label: "Node type"
     help: |
-      - **any** - (*12 cores*) Chooses anyone of the available Oakley nodes.
-        This reduces the wait time as you have no requirements.
-      - **vis** - (*12 cores*) This node includes an NVIDIA Tesla M2070 GPU
+      - **any** - (*28 cores*) Use any available Owens node. This reduces the
+        wait time as there are no node requirements.
+      - **vis** - (*28 cores*) This node includes an NVIDIA Tesla P100 GPU
         with an X server running in the background. This allows for Hardware
         Rendering with the GPU typically needed for 3D visualization using
-        VirtualGL. There are currently only 128 of these nodes on Oakley.
-      - **bigmem** - (*12 cores*) This Oakley node comes with 192GB of
-        available RAM. There are only 8 of these nodes on Oakley.
-      - **hugemem** - (*32 cores*) This Oakley node has 1TB of available RAM as
-        well as 32 cores. There is only 1 of these nodes on Oakley. A
-        reservation may be required to use this node.
+        VirtualGL. There are currently only 160 of these nodes on Owens.
+      - **hugemem** - (*48 cores*) Use an Owens node that has 1.5TB of
+        available RAM as well as 48 cores. There are 16 of these nodes on
+        Owens.
     options:
-      - ["any", ":ppn=12"]
-      - ["vis", ":ppn=12:vis:gpus=1"]
-      - ["bigmem", ":ppn=12:bigmem"]
-      - ["hugemem", ":ppn=32:hugemem"]
+      - ["any", ":ppn=28"]
+      - ["vis", ":ppn=28:vis:gpus=1"]
+      - ["hugemem", ":ppn=48:hugemem"]
   version:
     widget: select
     label: "COMSOL Multiphysics version"
     help: "This defines the version of COMSOL you want to load."
     options:
-      - ["5.1", "comsol/51"]
       - ["5.2a", "comsol/52a"]
```

Comment:
Probably don't need a `select` widget if there is only one version available. This can be directly hardcoded with:

```yaml
version: "comsol/52a"
```

Reply:
More versions will be added later. There's no sense removing it now.

Reply:
I'd remove it for now as it appears odd for a single selection (you can comment it out if you want to make it easier to add back in).

There is some precedence for not having a select field with a single version of the app:

- https://github.com/OSC/bc_osc_ansys_workbench
- https://github.com/OSC/bc_osc_matlab
- https://github.com/OSC/bc_osc_paraview
- https://github.com/OSC/bc_osc_jupyter
- https://github.com/OSC/bc_osc_jupyter_spark
- https://github.com/OSC/bc_osc_rstudio_server

Reply:
I can look at those but I was avoiding it at the time of the PR because this configuration file did not appear to be well-documented. I'll look into the ood-documentation repo to see if I can find reference to it.